server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/tasks/TaskFromRequestRegistry.java

```diff
@@ -171,10 +175,8 @@ public TaskHandler fromRequest(Request request) throws Exception {
                 + ". " + supportedValueMessage()));
     }
 
-    @Override
-    public Route asRoute(TaskManager taskManager) {
-        return new MultiTaskRoute(this, taskManager); //temporary, to test...
-        // TODO: need something to differentiate a normal task route from a multi one... need the request? or need a dedicated TaskFromRequestRegistry?
+    public Route asMultiRoute(TaskManager taskManager) {
+        return new MultiTaskRoute(this, taskManager);
```

Comment:
Dont like this but didnt find better... Not sure how to differentiate the difference of route here in the TaskFromRequestRegistry